src/config/watcher.rs

```diff
@@ -4,7 +4,10 @@ use std::{
     thread,
 };
 
-use notify::{recommended_watcher, EventKind, RecommendedWatcher, RecursiveMode, Watcher};
+use notify::{
+    recommended_watcher, EventKind, PollWatcher, RecommendedWatcher, RecursiveMode,
+    Watcher as NotifyWatcher,
```

Comment:
Why this rename? I think it would be better to normalize create custom aliases as much as possible. In this case, there is no conflict with that trait name

Reply:
Oh I see you have a `Watcher` enum.. I don't know if we could find a better name for it and not use an alias.. maybe specifying the notify watcher with `notify::Watcher`?